Add Leader Election to our website (#316)
strimzi/strimzi.github.io: _posts/2022-09-13-leader-election.md

---
layout: post
title:  "Video: Leader Election - Running Cluster Operator with multiple replicas"
date: 2022-09-13
author: jakub_scholz
---

Leader Election is one of the new features in Strimzi 0.31.0.

> In our new video, we look at it in more detail.

Reviewer: ```suggestion
In our new video, we look at the feature in more detail.
```

> We explain how it works, when is it useful and show a demo of Leader Election in action.

Reviewer 2: "when it is useful" ... and "shows a demo" ?

Author: Why _shows a demo_? It is _We show a demo_ so I think that is correct.

Author: @PaulRMellor Can you please have a look?

Reviewer 2: No no you are right :-D ... I read as "it shows", my bad. Only thin I am not sure if for a native English it needs a reinforcement of "We" again because the verb is so far from the subject. But it's something for @PaulRMellor :-D

Reviewer: Let's just use demo as a verb ;-)

> We explain how it works, when is it useful and show a demo of Leader Election in action.

Reviewer: ```suggestion
We explain how leader election works, when it's useful, and demo Leader Election in action.
```
